scripts/show-features.py

```diff
@@ -113,9 +115,12 @@ def main(argv=None):
         logger.error("%s", str(e))
         return -1
 
+    dynamic = (args.process) or (args.format == "cape") or (os.path.splitext(args.sample)[1] in ("json", "json_"))
```

Comment:
use `EXTENSIONS_CAPE` here?

Reply:
```suggestion
    is_dynamic = (args.process) or (args.format == "cape") or (os.path.splitext(args.sample)[1] in ("json", "json_"))
```

use variable name prefix `is_` for booleans